scripts/accumulator-server.py

```diff
@@ -373,46 +398,76 @@ def record_2871():
     else:
         # Ad hoc response related with issue #2871, see https://github.com/telefonicaid/fiware-orion/issues/2871
         r = Response(status=200)
-        r.data = '{"contextResponses":[{"contextElement":{"attributes":[{"name":"turn","type":"string","value":""}],"id":"entity1","isPattern":false,"type":"device"},"statusCode":{"code":200,"reasonPhrase":"OK"}}]}'
+        r.data = '''
+            {
+              "contextResponses": [
+                {
+                  "contextElement": {
+                    "attributes": [
+                      {
+                        "name": "turn",
+                        "type": "string",
+                        "value": ""
+                      }
+                    ],
+                    "id": "entity1",
+                    "isPattern": false,
+                    "type": "device"
+                  },
+                  "statusCode": {
+                    "code": 200,
+                    "reasonPhrase": "OK"
+                  }
+                }
+              ]
+            }
+        '''
         return r
 
 # Next 6 ones are for testing subscription status and failure logic. They are used by test
 # 1126_GET_v2_subscriptions/lastsuccesscode_and_lastfailurereason.test
 
+
 @app.route("/giveme200", methods=['POST'])
 def giveme200():
     return Response(status=200)
 
+
 @app.route("/giveme400", methods=['POST'])
 def giveme400():
     return Response(status=400)
 
+
 @app.route("/giveme404", methods=['POST'])
 def giveme404():
     return Response(status=404)
 
+
 @app.route("/giveme500", methods=['POST'])
 def giveme500():
     return Response(status=500)
 
+
 @app.route("/givemeDelay", methods=['POST'])
 def givemeDelay():
     sleep(60)
     return Response(status=200)
 
+
 @app.route("/waitForever", methods=['POST'])
 def waitForever():
     sleep(6000000000)    # Arround 20 years.. close enough to "forever" :)
     return Response(status=200)
 
+
 @app.route('/dump', methods=['GET'])
 def dump():
     return ac
 
 
 @app.route('/times', methods=['GET'])
-def times():
-    return ', '.join(map(str,times)) + '\n'
+def times_():
```

Comment:
`times` is a new reserved word in Python3?

Reply:
No, but `times` as variable is frequently used along the file, so python3 interpreter has some conflicts when some function name collides with a variable name. Since this function is just used to implement the endpoint and it's not ever called ... I decided just to rename the function :thinking:

Reply:
```suggestion
def get_times():
```

I think that's a better name :)

Reply:
What about:

```suggestion
def giveTimes():
```
?

Reply:
It seems we have a crazy mess of [snake_case](https://en.wikipedia.org/wiki/Snake_case) (dict_raise_on_duplicates, record_request, ... ) and [camelCase](https://en.wikipedia.org/wiki/Camel_case) (waitForever, givemeDelay, ...) names for functions in the code :) so both options are ok for me. Whichever you prefer.

Reply:
I think I prefer the camelCase for this ... **time** :rofl:

Reply:
Fixed in 9fa9d1a453cdd84d9b9fbcde1f73839cafca0225